### DISCUSSION TITLE
API Operation - @x-codeSample

### DISCUSSION THREAD

**Discussion by Remu5** [swaggo/swag#1280] [Q&A]
Does anyone have an example of how this x-codeSample works? Everytime it says I need a valid json value but Im not sure what that would look like.

**Comment by git-az** [comment]
This can work:

// @x-codeSamples file
// @summary CreateOrder

`swag` will search file  CreateOrder.json in dir (`--cef dir`)

**Comment by git-az** [comment]
// @x-codeSample {"lang":"golang", "source":"log.Printf(\"hello world\\n\")"}

`@x-codeSample jsonstring` vs `@x-codeSamples file`
